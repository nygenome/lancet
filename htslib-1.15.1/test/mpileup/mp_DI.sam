@HD	VN:1.6	SO:coordinate
@SQ	SN:z	LN:13
@CO	
@CO	Copyright (c) 2018 Genome Research Ltd.
@CO	
@CO	Permission is hereby granted, free of charge, to any person obtaining
@CO	a copy of this software and associated documentation files (the
@CO	"Software"), to deal in the Software without restriction, including
@CO	without limitation the rights to use, copy, modify, merge, publish,
@CO	distribute, sublicense, and/or sell copies of the Software, and to
@CO	permit persons to whom the Software is furnished to do so, subject
@CO	to the following conditions:
@CO	
@CO	The above copyright notice and this permission notice shall be included
@CO	in all copies or substantial portions of the Software.
@CO	
@CO	THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
@CO	OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
@CO	MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
@CO	IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
@CO	CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
@CO	TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
@CO	SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
@CO	
@CO	               1
@CO	    123  4567890  123 Depadded base numbering
@CO	ref TAG  CTTAGCA  GGT
@CO	
@CO	s1   AG  CTTAG**TTGGC 7M 2D 2I 3M
@CO	s2   AG  CTTAG**TT    7M 2D 2I
@CO	s4   AG  CTTAG***T    7M 2D 1P 1I
@CO	s3   **AACTTAGCA  GG  2D 2I 9M
@CO	s5   ***ACTTAGCA  GG  2D 1P 1I 9M
@CO	     01AB2345678AB901
@CO	
s1	0	z	2	0	7M2D2I3M	*	0	0	AGCTTAGTTGGC	0123456AB901
s2	0	z	2	0	7M2D2I	*	0	0	AGCTTAGTT	0123456AB
s4	0	z	2	0	7M2D1P1I	*	0	0	AGCTTAGT	0123456B
s3	0	z	2	0	2D2I9M	*	0	0	AACTTAGCAGG	AB234567890
s5	0	z	2	0	2D1P1I9M	*	0	0	ACTTAGCAGG	B234567890
