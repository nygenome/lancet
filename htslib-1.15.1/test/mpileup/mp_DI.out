z	2	5	^!A^!A^!A^!*^!*
z	3	5	GGG*+2(AA)*+2(*A)
z	4	5	CCCCC
z	5	5	TTTTT
z	6	5	TTTTT
z	7	5	AAAAA
z	8	5	G-2()G-2()G-2()GG
z	9	5	***CC
z	10	5	*+2(TT)*+2(TT)$*+2(*T)$AA
z	11	3	GGG
z	12	3	GG$G$
z	13	1	C$
